### PR TITLE
Feat/image version

### PR DESCRIPTION
**Pull request by tobi-bams**
Current Image version a service is running on is displayed on the side bar

### CODE REVIEW

File: app/src/Dashboard.svelte [hunk 1]
```diff
@@ -21,12 +21,59 @@
...
+          `sphinxlightning/${image_name}`
+        );
+        if (image_digest_response.success) {
+          const version = await getVersionFromDigest(
+            image_digest_response.digest,
+            `https://hub.docker.com/v2/repositories/sphinxlightning/${image_name}/tags?page_size=100`
+          );
+
+          console.log(image_name);
+          console.log(version);
```
Comment (kevkevinpal) [inline app/src/Dashboard.svelte:59]: we can delete these

**Review by kevkevinpal — COMMENTED** [review]
can you post a screenshot of what it'd look like?

**Comment by tobi-bams** [comment]
> can you post a screenshot of what it'd look like?

Sure!!

**Comment by tobi-bams** [comment]
Screenshot 2024-05-07 at 17 29 26
Screenshot 2024-05-07 at 17 29 31
Screenshot 2024-05-07 at 17 29 37
Screenshot 2024-05-07 at 17 29 45
Here are some Screenshots

**Pull request merged**